src/blop/integrations/ax/agent.py

```diff
@@ -59,8 +62,8 @@ class AxAgent:
         The degrees of freedom that the agent can control, which determine the output of the model.
     objectives : list[Objective]
         The objectives which the agent will try to optimize.
-    db : Broker
-        The databroker instance to read back data from a Bluesky run.
+    db : databroker or tiled
```

Comment:
Give the object types here:
- `Broker`
- `Container` (I think)
```suggestion
    db : `Broker | Container`
```